.github/workflows/publish.yml

```diff
@@ -22,6 +20,20 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Read package names from sdk.txt and extra_docs.txt
+        id: construct-packages
+        run: |
+          PACKAGES=$(awk '{print $1}' sdk.txt extra_docs.txt | tr '\n' ' ' | sed 's/ $//')
```

Comment:
This results in installing latest versions of packages later (`for package in $PACKAGES; do...`); the idea was to install the packages of the same version as in `mirror.yml`.